ecs-cluster-infrastructure-service-alb-global-accelerator.tf

```diff
@@ -0,0 +1,67 @@
+resource "aws_globalaccelerator_accelerator" "infrastructure_ecs_cluster_service_alb" {
+  count = local.infrastructure_ecs_cluster_services_alb_enable_global_accelerator ? 1 : 0
+
+  name            = "${local.resource_prefix}-infrastructure-ecs-cluster-service-alb"
+  ip_address_type = "DUAL_STACK"
```

Comment:
do we want to output the ips to the infrastructure map?

Reply:
Yep I need to play catchup with the resources we've added ... There's quite a few to be added in there now, I'll put that in as a new PR